src/Assets/otherProjects.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
I think we got some great inspiration during demon on how to solve the issues with two json-files, but I think this is an okay solution as well :)